reco_utils/docker/pyspark/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+ARG BASE_CONTAINER=jupyter/pyspark-notebook
+FROM $BASE_CONTAINER
+
+LABEL maintainer="Microsoft Recommender Project <RecoDevTeam@service.microsoft.com>"
+
+ARG WORK_PATH=/home/$NB_USER/work
+ARG RECO_PATH=$WORK_PATH/Recommenders
+
+USER root
+
+# Clone repo & discard notebooks that cannot be run in this environment
+RUN git clone --single-branch --branch only_pyspark https://github.com/Microsoft/Recommenders.git && \
```

Comment:
it seems like retaining seperate branches for notebooks that are relevant for different environments will be hard to maintain. Do we need to remove them? Right now we're assuming the user understands and has set up the correct environment (and ideally that environment is documented clearly in each notebook - this may not be the case but should be)